src/extensions/default/bramble/lib/PostMessageTransport.js

```diff
@@ -223,7 +224,7 @@ define(function (require, exports, module) {
     var _pendingReloadUrl;
 
     // Whether or not to allow reloads in the general case (true by default)
-    var _autoUpdate = true;
+    var _autoUpdate = PreferencesManager.get("livePreviewAutoReload");;
```

Comment:
This won't do what you want.  Here, you're getting the value of the pref when the module is loaded.  But then, during the lifetime of the app, you'll never go back and get it again.  What you should do instead is dump this variable altogether, and instead use `PreferencesManager.get("livePreviewAutoReload")` whenever the other variable was used.  As far as the `setAutoUpdate(value)` function, you can change it to call `PreferencesManager.set("livePreviewAutoReload", value)`.  So basically, we want to move this variable out of this module, and instead have it only live in one place: the preference manager.